bin/install

```diff
@@ -54,13 +54,13 @@ Options:
         set -g OMF_CHANNEL dev
 
       case '--channel=*'
-        abort "Unknown release channel \""(echo "$argv[1]" | command cut -d= -f2)"\"."
+        abort "Unknown release channel \""(echo "$argv[1]" | command cut -d= -f1 --complement)"\"."
```

Comment:
Can you please show a working example of `cut -d= -f1 --complement`?

I tried with `echo --output=foo=bar | command cut -d= -f1 --complement` which gives the following error:

```
cut: illegal option -- -
usage: cut -b list [-n] [file ...]
       cut -c list [file ...]
       cut -f list [-s] [-d delim] [file ...]
```

Reply:
Apparently the ```--complement``` options is [only available in GNU cut](https://www.gnu.org/software/coreutils/manual/html_node/cut-invocation.html) and not POSIX. My bad, I am surry, I used the help page to find a solution.

The POSIX compatible way is:

~~~
$ echo --output=foo=bar | command cut -d= -f2-
foo=bar
~~~

Thanks for catching this!